src/management/ManagementTokenProvider.js

```diff
@@ -71,6 +71,40 @@ var ManagementTokenProvider = function(options) {
     clientInfo: this.options.clientInfo
   };
   this.authenticationClient = new AuthenticationClient(authenticationClientOptions);
+
+  var self = this;
+  this.getCachedAccessToken = Promise.promisify(
```

Comment:
would this be the same as exporting it this way `ManagementTokenProvider.prototype.getCachedAccessToken`?